src/Components/SmartComponents/Modals/CvePairStatusModal.js

```diff
@@ -88,12 +88,21 @@ export const CvePairStatusModal = ({ cves, updateRef, inventories, hasDifferentS
     }
 
     function getCvestatus() {
-        if (cveList) {
-            if (checkboxState && !hasDifferentStatus) { return cveList[0].cve_status_id.toString();}
-            else if (hasDifferentStatus) {return '0';}
-            else if (!checkboxState && !hasDifferentStatus) {return cveList[0].status_id.toString();}
-            else {return '0';}
-        } else {return '0';}
+        switch (type) {
+            case 'systemsExposed' : {
+                return (cveList && cveList.length === 1 && cveList[0].status_id.toString()) || '0';
+            }
+
+            case 'systemDetail': {
+                if (cveList) {
+                    if (checkboxState && !hasDifferentStatus) { return cveList[0].cve_status_id || 0;}
+                    else if (hasDifferentStatus) {return '0';}
+                    else if (!checkboxState && !hasDifferentStatus) {return cveList[0].status_id;}
+                    else {return '0';}
+                } else {return '0';}
+            }
```

Comment:
it always a good practice when using a switch  to provide a default case when none of the values match the value of the expression (`type`).  for this default case `return: '0'` would be okay

Reply:
I am quite good at making too many mistakes )). I solved the issue, can you please have a look.

Reply:
that's not a mistake :)